src/diagnostics.ts

```diff
@@ -2,9 +2,9 @@ import * as ts from "typescript";
 import { TranspileError } from "./TranspileError";
 
 export const transpileError = (error: TranspileError): ts.Diagnostic => ({
-    file: error.node.getSourceFile(),
-    start: error.node.getStart(),
-    length: error.node.getWidth(),
+    file: error.node.kind === ts.SyntaxKind.Unknown ? undefined : error.node.getSourceFile(),
```

Comment:
I wonder what error required that change?

Reply:
It is because of `TSTLErrors.LuaEntryNotFound`. A TranspilerError which requires a node which isn't available when encountering this error